index.html

```diff
@@ -322,6 +322,11 @@ <h2>
             </li>
           </ol>
         </li>
+        <li>Otherwise, if <var>manifest</var>["<a>start_url</a>"] is not
+        present, or not <a>within scope</a> of the {{Document}}'s
+          <a data-cite="DOM#dom-document-url">URL</a>, consider the document
+          not <a>installable</a>.
+        </li>
```

Comment:
Is "in scope of a URL" defined? What does that mean for https://example.com/a/b/c/page.html?

Reply:
There's a section in the navigation-scope section of the spec defining in scope. https://www.w3.org/TR/appmanifest/#navigation-scope

Reply:
Yeah, that behaviour doesn't sound appropriate here. Would be good to have test cases for this (input -> output examples).

Reply:
Oops. It's been awhile since I wrote this so I'm not sure what my intention was, but I suspect this was simply a mistake. It should have said "not same origin as", not "not within scope of".

That matches the existing check (which I'm removing), that would check if start_url was same origin as the document URL, and if not, ignore the supplied start_url and default to the document URL. Only now, we just consider it non-installable.

Does that sound right?

Reply:
That sounds right.